core/layouts/simmons/simmons.info

```diff
@@ -0,0 +1,29 @@
+name = Simmons
+version = BACKDROP_VERSION
+backdrop = 1.x
+type = layout
+
+; Specify regions for this layout.
+regions[header]       = Header
+regions[top]          = Top
+regions[content]      = Content
+regions[sidebar]      = First Sidebar
+regions[sidebar2]     = Second Sidebar
+regions[third1]       = Tryptych First
+regions[third2]       = Tryptych Second
+regions[third3]       = Tryptych Third
+regions[quarter1]     = Quarter First
+regions[quarter2]     = Quarter Second
+regions[quarter3]     = Quarter Third
+regions[quarter4]     = Quarter Fourth
```

Comment:
Okay after discussing, let's keep the current naming convention. However let's still fix the spelling in the Triptych label.